test/opf-var.jl

```diff
@@ -127,8 +127,8 @@ end
             result = PowerModels._solve_opf_cl(data, SDPWRMPowerModel, sdp_solver)
 
             @test result["termination_status"] == OPTIMAL
-            @test isapprox(result["objective"], 5728.62; atol = 1e0)
-            #@test isapprox(result["objective"], 5747.63; atol = 1e0)
+            # @test isapprox(result["objective"], 5728.62; atol = 1e0)
+            @test isapprox(result["objective"], 5747.63; atol = 1e0)
```

Comment:
I don't know what drove this, but the new optimal values were already here and commented out?

Reply:
Nothing to be alarmed about the SDP solution are super unstable.